### DISCUSSION TITLE
Install latest version of Pybricks library

### DISCUSSION THREAD

**Discussion by Xylar52** [pybricks/support#1045] [Q&A] [answered]
I want to code spike prime in vscode, and i want the library to be up-to-date. I was told to install pybricks library. However, https://pypi.org/project/pybricks/ says that the library has not been updated since december of 2022. So i tried to install the Pybricks-Micropython library directly with the github link and it didn't work. How can i update the library to have access to the latest add, especially the imu

**Comment by laurensvalk** [comment, marked as answer]
It defaults to the latest stable version, but you could pick the latest pre-release. (This is from the link you posted.)

So instead of installing `pybricks`, you could install `pybricks==3.3.0a4`.

![image](https://user-images.githubusercontent.com/12326241/234242793-56bf53e8-7cdd-4320-97db-727aa1797d07.png)
